aquarius/graphql.py

```diff
@@ -81,6 +81,37 @@ def get_number_orders_price(token_address, last_sync_block, chain_id):
         return -1, {}
 
 
+def get_nft_transfers(start_block, last_sync_block, chain_id):
+    try:
+        client = get_client(chain_id)
```

Comment:
I think this is duplicated with the code in get_number_orders. Could we extract the waiting part to avoid duplication?